lib/studies/shrine/expanding_bottom_sheet.dart

```diff
@@ -556,7 +556,6 @@ class _ExpandingBottomSheetState extends State<ExpandingBottomSheet>
       key: _expandingBottomSheetKey,
       duration: const Duration(milliseconds: 225),
       curve: Curves.easeInOut,
-      vsync: this,
```

Comment:
Would this class still need to be a `TickerProviderStateMixin`

Reply:
Removed the `TickerProviderStateMixin`